app/domain/errors.rb

```diff
@@ -449,6 +449,11 @@ module Constraints
         code: "CONJ00069E"
       )
 
+      RoleMissingAnyConstraints =  ::Util::TrackableErrorClass.new(
+        msg: "Role must have at least one constraint",
+        code: "CONJ00070E"
```

Comment:
@shulifink

Reply:
@tzheleznyak What is a Role? does the user will understand this massage?

Reply:
The role here in the policy 
```
- !host
  id: aaa
- !grant
  role: !group conjur/authn-jwt/aaa/aaa
  member: !user aaa
```

Reply:
When is this disopayed? What are the constraints?

Reply:
Fixed to at least one relevant restriction